peercrawler.py

```diff
@@ -82,6 +82,34 @@ def count_good_peers(self):
     def count_peers(self):
         return len(self.get_peers_copy())
 
+    def listen_incoming(self):
+        with socket.socket(socket.AF_INET6, socket.SOCK_STREAM) as s:
+            s.bind(("::", 7075))
+            s.listen()
+
+            while True:
+                connection, address = s.accept()
+                threading.Thread(target=self.handle_incoming, args=(connection, address), daemon=True).start()
+
+    def handle_incoming(self, connection: socket.socket, address):
```

Comment:
This will work but I think it might be better to close non realtime connections straight away because they will never gives us a keepalive packet. And by habing them open, we are just wasting resources on both sides until something timeosout.

Reply:
To make sure I'm understanding this correctly, you want to have only a single incoming connection open at a time?

Reply:
No, I would like to have X number of incoming connections in parallel, where X is configurable.

Reply:
There 2 types of sockets, realtime and bootstrap.
Only realtime connections will give us a keepalive.
A bootstrap will never give us a keepalive.
So if it a bootstrap, close it straight away.
Bootstrap connections do not do a handshake.